admin/users/serializers.py

```diff
@@ -0,0 +1,29 @@
+"""
+Serialize user
+"""
+# from admin.nodes.serializers import serialize_simple_user
+
+
+def serialize_user(user):
+    two_factor = user.get_addon('twofactor')
+    if two_factor is None or two_factor is False:
```

Comment:
Is it `None` if TF has never been configured and `False` if it previously was? If not, are both of these cases necessary?

Alternatively, APIv2 has [helper methods](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/utils.py#L17-L33) to determine [truth/fals]yness, it might be good to implement or reuse something like that for ADM.